smarts/core/mission_planner.py

```diff
@@ -277,11 +282,60 @@ def cut_in_waypoints(self, sim, pose: Pose, vehicle):
     def uturn_waypoints(self, sim, pose: Pose, vehicle):
         # TODO: 1. Need to revisit the approach to calculate the U-Turn trajectory.
         #       2. Wrap this method in a helper.
-        radius = self._mission.task.trigger_radius
         neighborhood_vehicles = sim.neighborhood_vehicles_around_vehicle(
-            vehicle=vehicle, radius=radius
+            vehicle=vehicle, radius=200
         )
 
+        if not neighborhood_vehicles:
+            return []
+
+        aggressiveness = self._agent_behavior.aggressiveness
+        if sim.elapsed_sim_time < 0.5:
+            return []
+        if sim.elapsed_sim_time == 0.5:
```

Comment:
We shouldn't rely on `elapsed_sim_time == 0.5`

Reply:
Thanks,